pages/TheExplorationGallery.html

```diff
@@ -11,32 +11,40 @@
   <p>
     After signing in to Oppia, you'll end up on the Home page. Even before you
     start creating your own explorations, there are a number of things you can
-    do. You can click on the icon at the top left corner of the page to access
-    the blog, join the discussion forum or access the Participation Playbook.
-    You can also check out a variety of existing explorations in different
-    categories, or create one of your own by clicking on "Create Exploration"
+    do. You can click on the about icon at the top right corner of the page 
+    (for large screen sizes) or the icon at the top left corner (for small 
+    screen sizes) to access the blog, join the discussion forum or access the
+    teach with oppia and other options. You can also check out a variety of existing explorations 
+    in different categories, or create one of your own by clicking on "Create Exploration" 
     at the top right.
   </p>
 
-  <img src="static/images/frontPage.png" width="100%">
+  <img src="static/images/frontPageLargeScreen.png" width="100%">
 
   <p>
-    Below the banner, the homepage contains a list of all published, non-draft
-    explorations on the server. If there are a lot of explorations, not all of
-    them will be shown at once. As you scroll down, more will be loaded.
+  	for other screen sizes
   </p>
 
-  <img src="static/images/gallery.png" width="100%">
+  <img src="static/images/frontPageSmallScreen.jpg" width="100%">
+
+  <p>
+    The LIBRARY contains a list of all published, non-draft explorations
+    on the server. The explorations are grouped in various categories that 
+    they best fit in. Example, the quadratic equation is found under the 
```

Comment:
"Example, ..." --> "For example, the exploration on quadratic equations can be found under the category 'Mathematics and Statistics'."